pyshoco.c

```diff
@@ -0,0 +1,42 @@
+#include <Python.h>
+#include <string.h>
+#include "shoco.h"
+
+static PyObject *pyshoco_compress(PyObject *self, PyObject *args) {
+    const char *in;
+
+    if (!PyArg_ParseTuple(args, "s", &in))
+        return NULL;
+    
+	const size_t bufsize = strlen(in)*2+1;

```

Comment:
My apologies, I'm still getting used to a new editor. I've fixed this in the latest commit.